### DISCUSSION TITLE
Genmon stopped working after six years

### DISCUSSION THREAD

**Discussion by shpegel** [jgyates/genmon#1107] [General]
My Genmon system has been up and running for nearly 6 years and has given almost no trouble.  Two months ago it just stopped responding although I could still SSH into the Pi. Reboots did not fix . It finally got warm enough to investigate and I traced the problem to a bad SD card. I kept an image backup so it was easy to get going again.  Just posting this in hopes it will help others.

A few other oddities which I mention but don't really see as real problems: After Genmon reboots, the browser screen does not update the screen and stays on rebooting screen. Chrome and Edge both act the same. Just open a new window after a brief wait and your good. Earthlink email account used to work for notifications no longer does. I just switched to my outlook.com account to fix.

**Comment by jgyates** [comment]
@shpegel 

Thanks for the info. The browser refresh is expected. There is no periodic refresh of the javascript except when a software update occurs so a manual refresh is required if you reboot your pi. 

If you have any error logs from /var/log/mymail.log or from the test email on the Settings page I may be able to offer some insight on the earthlink issue, otherwise I will make a note about earthlink email compatiblity.

**Reply by jgyates** [reply]
I also updated the title of the post to highlight the long duration of genmon running :)
